IdentityCore/src/oauth2/MSIDTokenResponse.m

```diff
@@ -126,6 +126,11 @@ + (MSIDProviderType)providerType
     @throw MSIDException(MSIDGenericException, @"Abstract method was invoked.", nil);
 }
 
+- (NSString *)accountIdentifier
+{
+    return nil;
```

Comment:
Should this be nil, or something reasonable that exists in OAuth2?

Reply:
Changed to self.idTokenObj.uniqueId